chain/lf3/participation.go

```diff
@@ -171,69 +191,49 @@ func (p *Participant) tryF3Participate(ctx context.Context, ticket api.F3Partici
 			p.backOff(ctx)
 			continue
 		case err != nil:
+			if p.backoff.Attempt() > float64(p.maxCheckProgressAttempts) {
```

Comment:
IMO, max check limits aren't really necessary anyways but... I _think_ we should have had one here?